addons/point_of_sale/static/src/app/screens/product_screen/control_buttons/control_buttons.xml

```diff
@@ -1,26 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="point_of_sale.ControlButtons">
-        <div t-attf-class="flex-{{props.wrapped ? 'wrap' : 'column'}}" class="control-buttons d-flex bg-300 border-bottom">
+        <div t-attf-class="flex-{{!props.wrapped ? (ui.isSmall ? 'column' : 'wrap') : 'wrap'}}" class="control-buttons d-flex bg-300 border-bottom">
+            <SelectPartnerButton partner="partner"/>
```

Comment:
Same as last time, why passing props since `SelectPartnerButton` have a direct access to the pos_store ?

I know that isn't your change but if you can get rid of that is cool. Is up to you.